queries/pois.jinja2

```diff
@@ -51,7 +51,13 @@ UNION ALL
     aerialway, aeroway, amenity, barrier, craft, highway, historic, leisure, lock, man_made, "natural", office, power, railway, shop, tourism, waterway,
     tags->'attraction' AS attraction,
     tags->'zoo' AS zoo,
-    NULL AS transit_routes,
+    -- note: the mz_calculate_transit_routes function is pretty expensive,
```

Comment:
Previously it was always `NULL`, so if anything the performance will get worse. But probably only a little bit, since this function isn't _that_ expensive and stations are pretty rare features.